From 007c25790f10f6ca9ad847bd186671876770d460 Mon Sep 17 00:00:00 2001
From: Frantisek Sumsal <frantisek@sumsal.cz>
Date: Mon, 15 Apr 2019 13:59:27 +0200
Subject: [PATCH 08/29] test: don't timeout while waiting for other test units

The main testsuite service timeouts sporadically when waiting for
other testsuite-* units. As the test timeout is handled by
the "test executor" (test.sh), let's disable it for the service.

This should (hopefully) fix the test flakiness.

(cherry picked from commit 772f15c1d8e717d9eb44e256a3c076598cdb2cea)
---
 test/TEST-16-EXTEND-TIMEOUT/testsuite.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/TEST-16-EXTEND-TIMEOUT/testsuite.service b/test/TEST-16-EXTEND-TIMEOUT/testsuite.service
index e1cd5caeea..7512ba9e12 100644
--- a/test/TEST-16-EXTEND-TIMEOUT/testsuite.service
+++ b/test/TEST-16-EXTEND-TIMEOUT/testsuite.service
@@ -12,7 +12,7 @@ Wants=testsuite-fail-runtime.service
 StopWhenUnneeded=yes
 
 [Service]
-
 Type=simple
+TimeoutStartSec=infinity
 ExecStartPre=/assess.sh
 ExecStart=/bin/true
-- 
2.21.0

